app/views/devise/shared/_error_messages.html.erb

```diff
@@ -1,15 +1,2 @@
 <% if resource.errors.any? %>
-  <div id="error_explanation">
```

Comment:
if you are not using this file just remove it, don't leave it with a empty if statement